internal/resources/sourcesecret/constant.go

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package sourcesecret
+
+const (
+	ResourceName     = "tanzu-mission-control_repository_credential"
```

Comment:
nit: new line just for better readability

Reply:
Updated